resources/views/checkout/steps/success.blade.php

```diff
@@ -1,68 +1,74 @@
 <checkout-success>
-    <div dusk="checkout-success">
+    <div dusk="checkout-success" slot-scope="{ order, hideBilling, shipping, billing, items }" v-cloak v-if="order">
         <h1 class="font-bold text-4xl mb-5">@lang('Order placed succesfully')</h1>
         <div class="bg-highlight rounded p-8">
             <p>@lang('We will get to work for you right away')</p>
-            <p>@lang('We will send a confirmation of your order to') <span class="font-bold">@{{ $root.user?.email }}</span></p>
+            <p>@lang('We will send a confirmation of your order to') <span class="font-bold">@{{ order.customer_email }}</span></p>
         </div>
         <div class="mt-4">
-            <div class="flex flex-wrap items-center mb-4 border-b pb-2" v-for="(item, productId, index) in cart.items">
+            <div
+                class="flex flex-wrap items-center mb-4 border-b pb-2"
+                v-for="(item, productId, index) in items"
+                v-if="!item.parent_item_id"
```

Comment:
Within the Vue component we're already filtering these out right?

Reply:
Oh whoops, I guess I forgot I'd already done that when I first found this fix 😅